### ISSUE TITLE
Crashlytics deobuscation no longer getting uploaded with Gradle Plugin 3.5.0

### ISSUE THREAD

**Issue by red42** [firebase/firebase-android-sdk#745]
### [REQUIRED] Step 2: Describe your environment

  * Android Studio version: 3.5.0
  * Firebase Component: Crashlytics
  * Component version: 
com.crashlytics.sdk.android:crashlytics:2.10.1
io.fabric.tools:gradle:1.26.1 - also 1.30.0

### [REQUIRED] Step 3: Describe the problem

The stack traces in Crashlytics on the Firebase Console no longer show deobfuscated code after upgrading Android Studio and Gradle plugin to version 3.5.0.

Same issue as [reported on Stack Overflow].(https://stackoverflow.com/questions/57498849/after-upgrading-to-gradle-5-4-1-and-android-stuidio-3-5-rc-3-crashlyticsuploadd)

The crashlyticsUploadDeobsRelease seems to be missing.

#### Steps to reproduce:

On Android Studio, run a build (Build/Generate Signed Bundle / APK), upload the bundle to Google Play. Open a Crashlytics report on Firebase Console. The stack trace is obfuscated.

Downgrading the Gradle plugin to 3.4.2 seems to solve the issue.

**Comment by xiaos** [comment]
same.

Android Studio version: 3.5.0
Firebase Component: Crashlytics
Component version:
com.crashlytics.sdk.android:crashlytics:2.10.1
io.fabric.tools:gradle: 1.31.0

**Comment by AndreiNikitin** [comment]
the same for me

Android Studio version: 3.5.0
Gradle plugin version: 3.5.0
Firebase Component: Crashlytics
Component version:
com.crashlytics.sdk.android:crashlytics: 2.9.4
io.fabric.tools:gradle: 1.28.0

**Comment by alikyo** [comment]
I fixed the issue, you have to upgrade the fabric gradle plugin
See here: https://docs.fabric.io/android/changelog.html#july-10-2019

**Comment by aguatno** [comment]
Hi @red42 sorry for my late response here. I found an internal bug for this, and it looks like the issue has been fixed in Fabric Gradle plugin [1.30.0](https://docs.fabric.io/android/changelog.html#july-10-2019). 

I am going to close this issue. If you're still experiencing this problem and want to continue the discussion just leave a comment here and we are happy to re-open this.

**Comment by lance0428** [comment]
It took me an embarrassing long time to figure out how to upgrade to the latest fabric gradle plugin for this fix.  For those that stumble upon this thread, it is define in your root build.gradle buildscript dependencies.
```
buildscript {
    dependencies {
        ...
        classpath 'io.fabric.tools:gradle:1.31.2'
        ...
    }
}
```
